examples/model_selection/plot_learning_curve.py

```diff
@@ -113,6 +113,8 @@ def plot_learning_curve(
         n_jobs=n_jobs,
         train_sizes=train_sizes,
         return_times=True,
+        random_state=0,
+        shuffle=True,
```

Comment:
```suggestion
```

The default should already be deterministic. The source of non-deterministic results is purely computational.